lib/ui/common/DogTagStyles.dart

```diff
@@ -21,7 +21,7 @@ class DogTagStyles {
             EdgeInsets.only(left: 12, top: 5.5, right: 12, bottom: 5.5),
         titleTextStyle: TextStyle(
             fontSize: 11,
-            fontWeight: FontWeight.w500,
+            fontWeight: FontWeight.w600,
```

Comment:
is this part of this PR?

Reply:
Totally agree, just saw that the designs are a little bit more bold in that dogTag and i dont realised that is out of scope of this PR, my bad